__tests__/utils/storage.spec.js

```diff
@@ -0,0 +1,23 @@
+const { Store } = require('data-store');
+const { getStoredValues, storeConfiguration } = require('../../src/utils/storage');
+
+describe('storage', () => {
+  const getMock = jest.fn();
+  const setMock = jest.fn();
+  beforeAll(() => {
+    jest.spyOn(Store.prototype, 'get').mockImplementation(getMock);
+    jest.spyOn(Store.prototype, 'set').mockImplementation(setMock);
+  });
+  describe('getStoredValues', () => {
+    it('should retrieve stored values', () => {
+      getStoredValues('test');
+      expect(getMock).toHaveBeenCalled();
+    });
+  });
+  describe('storeConfiguration', () => {
+    it('should save values', () => {
+      storeConfiguration('test', 123);
+      expect(setMock).toHaveBeenCalled();
```

Comment:
I think we need a bit more explicit testing here

Reply:
I didn't want to test too much here because these functions are essentially thin wrappers around the data-store package. We would be testing the package more than our own code